charts/cofide-connect/templates/deployment.yaml

```diff
@@ -8,20 +8,15 @@ spec:
   replicas: {{ .Values.replicaCount }}
   selector:
     matchLabels:
-      app.kubernetes.io/name: {{ .Release.Name }}
-      {{- include "cofide-connect.selectorLabels" . | nindent 6 }}
```

Comment:
Looks like we've lost the ability to customise labels?

Reply:
I've moved over to use the selector labels helper throughout as this was double-defining the name